docs/docs/sdk/prediction-import/index.md

```diff
@@ -23,7 +23,7 @@ from encord_active.lib.project import Project
 
 project_path = Path("/path/to/your/project/root")
 
-predictions: List[Prediction] = ...  # Your list of predictions
```

Comment:
I would keep `List` and then add 
```python
from typing import List
```
in the top of the block.

Reply:
We can use safely use `list` because `typing.List` (and a few others hints from that package) are deprecated since 3.9 and `list` is now the recommended type hint.

Reply:
Hmm okay - we use `List` everywhere in the code base, which is why I suggested it. Anyway, it's fine then.